src/components/views/project/ProjectIndex.tsx

```diff
@@ -258,19 +267,58 @@ const ProjectIndex: FC<IProjectBySlug> = () => {
 					{activeTab === 2 && <ProjectDonations />}
 					{activeTab === 3 && <ProjectGIVPowerIndex />}
 					{isDraft && (
-						<Flex $justifyContent='flex-end'>
-							<ContinueCreationButton
-								label={formatMessage({
-									id: 'label.continue_creation',
-								})}
-								buttonType='primary'
-								type='submit'
-								onClick={() =>
-									router.push(
-										idToProjectEdit(projectData?.id || ''),
-									)
-								}
-							/>
+						<Flex
+							$justifyContent='flex-end'
+							style={{ position: 'relative' }}
+						>
+							{/* Show tooltip only when the button is disabled (email not verified) */}
+							{!isEmailVerifiedStatus && (
+								<TooltipWrapper
+									style={{
```

Comment:
@maryjaf you should move style definition to style code bellow. 
Check this code example how to handle variable inside style:

https://github.com/Giveth/giveth-dapps-v2/blob/2981622542de1680f51aaa20588216be13414312/src/components/views/project/projectActionCard/AdminActions.tsx#L273